code-validation/rustfmt.toml

```diff
@@ -0,0 +1,3 @@
+use_field_init_shorthand = true
+newline_style = "Unix"
+format_code_in_doc_comments = true
```

Comment:
This is *very* nice. Once we get this working, we should transfer this to the main repo; this has been a frustration of mine.